examples/google.events.cloud.pubsub.v1.md

```diff
@@ -0,0 +1,64 @@
+## google.events.cloud.pubsub.v1
```

Comment:
This doesn't look like a Pub/Sub example - this is a CAL event.

Reply:
(Whereas the audit one *is* a Pub/Sub event - basically the files are just the wrong way round.)

Reply:
Fixed this.